com.biglybt.BiglyBT.yaml

```
@@ -0,0 +1,52 @@
+app-id: com.biglybt.BiglyBT
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: biglybt-runner
+separate-locales: false
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --persist=.biglybt
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: biglybt
+    buildsystem: simple
+    build-commands:
+      - mv biglybt /app/biglybt
+      - install -dm755 /app/bin
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/512x512/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/256x256/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/128x128/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/64x64/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/32x32/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/16x16/apps/com.biglybt.BiglyBT.svg
```

```suggestion
      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/scalable/apps/com.biglybt.BiglyBT.svg
```

One doesn't simply install SVG for "sized" icons.

---

Thanks for your rapid review and feedback, not sure how to go about the SVG? any hints?